apps/pie-docs/src/components/checkbox/overview/code.md

```diff
@@ -0,0 +1,138 @@
+    props: "{% include './props.json' %}"
+    slots: "{% include './slots.json' %}"
+    events: "{% include './events.json' %}"
+---
+
+## Overview
+
+<p>
+  <a href="https://www.npmjs.com/@justeattakeaway/pie-checkbox" style="text-decoration: none">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-checkbox.svg?label=pie-checkbox">
+  </a>
+
+  <a href="https://www.npmjs.com/package/@justeattakeaway/pie-webc">
+    <img alt="GitHub Workflow Status" src="https://img.shields.io/npm/v/@justeattakeaway/pie-webc.svg?label=pie-webc">
+  </a>
+</p>
+
+`pie-checkbox` is a Web Component built using the Lit library.
+
+This component can be easily integrated into various frontend frameworks and customized through a set of properties.
+
+## Installation
+
+To install `pie-checkbox` in your application, run the following on your command line:
+
+```shell
+npm i @justeattakeaway/pie-webc
+```
+
+```shell
+yarn add @justeattakeaway/pie-webc
+```
+
+## Peer Dependencies
+
+When using `pie-checkbox`, you will also need to include a couple of dependencies to ensure the component renders as expected. See [the PIE Wiki](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components#expected-dependencies) for more information and how to include these in your application.
+
+{% notification {
+  type: "neutral",
+  iconName: "link",
+  message: "For more information on using PIE components as part of an application, check out the [Getting Started Guide.](https://github.com/justeattakeaway/pie/wiki/Getting-started-with-PIE-Web-Components)."
+} %}
+
+## Playground
+
+ <iframe
+  src="https://webc.pie.design/?path=/story/checkbox--default&viewMode=story&shortcuts=true&singleStory=true"
+  width="100%"
+  height="600px"
+  style="border: none; margin-top: var(--dt-spacing-f);"
+></iframe>
+
+## Props
+
+{% componentDetailsTable {
+  tableData: props
+} %}
+
+## Slots
+
+{% componentDetailsTable {
+  tableData: slots
+} %}
+
+## Events
+
+{% componentDetailsTable {
+  tableData: events
+} %}
+
+## Importing and usage in templates
+
+For Native JS Applications:
+```js
+// import as module into a js file e.g. main.js
+import '@justeattakeaway/pie-webc/components/checkbox.js'
+```
+
+```html
+<!-- pass js file into <script> tag -->
+<pie-checkbox name="mycheckbox">Label</pie-checkbox>
+<script type="module" src="/main.js"></script>
+```
+
+For Vue Applications:
+
+```js
+// import as module into a js file that will be loaded on the page where the component is used.
+import '@justeattakeaway/pie-webc/components/checkbox.js';
+```
+
+```html
+<pie-checkbox name="mycheckbox">Label</pie-checkbox>
+```
+
+For React Applications:
+
+```jsx
+import { PieCheckbox } from '@justeattakeaway/pie-webc/react/checkbox.js';
+
+<PieCheckbox name="mycheckbox">Label</PieCheckbox>
+```
+
+```jsx
+// React templates (using Next 13 and SSR)
+import { PieCheckbox } from '@justeattakeaway/pie-checkbox/dist/react';
```

Comment:
We don't really explain why we are using a different package here. We will shortly have a ticket to unify what we put in these pages. But for now, please could we add some info on using the webc package vs the individual package? Otherwise I can see this being a little confusing for consumers

Reply:
I've shared some things I think that should be unified across the code pages with @maledr5 this is one of them. If it's ok can we tackle all examples like this in on go so it's all in sync please?